nni/retiarii/nn/pytorch/nasbench101.py

```diff
@@ -304,6 +304,8 @@ def make_list(x): return x if isinstance(x, list) else [x]
             [op_candidates[selected[f'{label}/op{i}']] for i in range(1, num_nodes - 1)],
             adjacency_list, in_features, out_features, num_nodes, projection)
 
+        # FIXME: weight inheritance on nasbench101 is not supported yet
```

Comment:
why nasbench101 is different?

Reply:
NAS-Bench-101 has a customized recipe to create a fixed cell, which includes removing unused nodes and reordering nodes to make the cell more compact. This makes supporting weight sharing itself on NAS-Bench-101 difficult, not to mention loading from super-net.

Reply:
so does NAS-Bench-101 support weight sharing?

Reply:
double confirm, does graph engine support weight sharing nas?

Reply:
> so does NAS-Bench-101 support weight sharing?

Currently, no.

> double confirm, does graph engine support weight sharing nas?

Graph engine and weight sharing are unrelated components. There's no such thing like "support".